src/software/parameter/config/ai_control.yaml

```diff
@@ -41,3 +41,8 @@ AIControlConfig:
     default: "Halt"
     description: >-
         Specifies the refbox play that should be in use
```

Comment:
This is slightly out-of-scope, but why do we call it a "refbox play" here and elsewhere?

Reply:
Good question. I'm trying to unify these things, so I'll change that. Also see #1551